### ISSUE TITLE
The SVG used as a breadcrumb arrow does not have alt text

### ISSUE THREAD

**Issue by jeremyfelt** [washingtonstateuniversity/medicine.wsu.edu#168]
Current HTML:

```
<img src="https://medicine.wsu.edu/wp-content/themes/wsu-medicine/images/ico-breadcrumb-arrow-sharp.svg">
```
